tests/integration/adapter-node-tests.js

```diff
@@ -4,35 +4,37 @@ export default function adapterNodeTests() {
   it('should handle recording requests posting a Buffer', async function() {
```

Comment:
Looks like we forgot to bring this over when we removed adapter-fetch's node builds in favor of recommending adapter-node-http over it.